File: EngagementHub/TOS.EngagementHub.Models/Employee.cs
using System;
using System.Collections.Generic;

namespace TOS.EngagementHub.Models
{
    public class Employee : EngagementModel
    {
        public Name Name { get; set; } = new Name();
        public DateTime? DateOfBirth { get; set; }
        public string Email { get; set; }
        public Guid? CityId { get; set; }
        public Guid? RankId { get; set; }
        public ICollection<EmployeeSkill> Skills { get; set; } = new List<EmployeeSkill>();
    }
}

File: NiceReads/Backend/Server/tos-server/src/application/models/Employee.ts
import { Uuid } from '../../common/uuid';
import { Document } from './Document';
import { EmployeeSkill } from './EmployeeSkill';
import { Name } from './Name';

export class Employee extends Document {
    constructor(
        public name: Name = new Name(),
        public dateOfBirth: Date | undefined = undefined,
        public email: string | undefined = undefined,
        public cityId: Uuid | undefined = undefined,
        public rankId: Uuid | undefined = undefined,
        public skills: EmployeeSkill[] = [],
        id: Uuid = Uuid.EMPTY
    ) {
        super(id);
    }
}